gufe/setup/network_planning/atom_mapping_based/atom_mapper.py

```diff
@@ -27,4 +27,5 @@ def suggest_mappings(self,
         Suggests zero or more :class:`.AtomMapping` objects, which are possible
         atom mappings between two :class:`.Component` objects.
         """
-        ...
+        raise NotImplementedError("This function was not implemented.")
```

Comment:
I believe this (...) is a pattern we use quite widely for our abcs?

Reply:
I thought we tried to get rid of this pattern, as you can break the code with some vodoo (found that some time ago)